aviary/variable_info/functions.py

```diff
@@ -146,11 +146,13 @@ def name_filter(name):
     if overridden_outputs:
         print("\nThe following variables have been overridden:")
```

Comment:
Add verbosity checks (only print if >= 1) to all print statements

Reply:
done.